Mathlib/GroupTheory/Index.lean

```diff
@@ -373,6 +373,15 @@ theorem relindex_eq_one : H.relindex K = 1 ↔ K ≤ H :=
 theorem card_eq_one : Nat.card H = 1 ↔ H = ⊥ :=
   H.relindex_bot_left ▸ relindex_eq_one.trans le_bot_iff
 
+@[to_additive]
+lemma inf_eq_bot_of_coprime {G : Type*} [Group G] {H K : Subgroup G}
+    (h : Nat.Coprime (Nat.card H) (Nat.card K)) : H ⊓ K = ⊥ :=
+  card_eq_one.mp (Nat.eq_one_of_dvd_coprimes h
+    (card_dvd_of_le inf_le_left) (card_dvd_of_le inf_le_right))
+
+@[deprecated (since := "2024-12-18")]
+alias _root_.add_inf_eq_bot_of_coprime := AddSubgroup.inf_eq_bot_of_coprime
```

Comment:
Could you also add an alias for the multiplicative version?

Reply:
I can't, because it would conflict with the new name when the namespace `Subgroup` is open